### PR TITLE
Add Option to display a user-friendly message on temporary lockout (fixes #17014)

### LINKED ISSUE #17014: Allow custom message for brute force temporary lockout
### Description

When a user account is temporarily locked out due to too many invalid login attempts, Keycloak returns the standard 'invalid username or password' error message to prevent leaking information.

The realm (or client) administrator should be able to substitute a custom message here.

### Discussion

_No response_

### Motivation

While the current behavior is the most secure behavior, it doesn't meet all business use cases. For many applications, particularly with consumers, returning a message such as "Your account has been temporarily locked out due to too many invalid logon attempts. Please try again in 15 minutes" may be a more appropriate response from the server.

The application owner should be allowed to make a tradeoff between security and usability based on his or her specific use case. Lowering support costs at the expense of leaking the existence of a user account ID, for example, may be an acceptable tradeoff. The application owner should be able to make this business decision.

### Details

_No response_

### PR DESCRIPTION
**Pull request by twobiers**
This change introduces a new setting on the Brute force protection to display a more transparent message to the user. 

Fixes #17014

### CODE REVIEW

File: js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties [hunk 1]
```diff
@@ -2773,6 +2773,7 @@ usersLeft_one={{count}} user left the group
 sync-keycloak-groups-to-ldap=Sync Keycloak groups to LDAP
 saveError=User federation provider could not be saved\: {{error}}
 bruteForceDetection=Brute force detection
+transparentUserMessage=Transparent to user
```
Comment (pedroigor) [inline js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties:2776]:
```suggestion
userFriendlyMessage=User-Friendly Message
```

**Comment by twobiers** [comment]
@pedroigor Thank you for your feedback. I agree, `user-friendly message` is a better description and have updated the files accordingly.

**Review by douglaspalmer — COMMENTED** [review]
There's a better way to handle this without introducing a new property. I will elaborate on the issue.

**Comment by danieleruatecnes** [comment]
OK thanks, I will follow the other open cases

**Pull request closed without merging**